app/models/case_log.rb

```diff
@@ -442,6 +442,29 @@ def is_dpo
     created_by&.is_dpo
   end
 
+  delegate :service_name, :sensitive, :registered_under_care_act, :primary_client_group, :has_other_client_group, :secondary_client_group, :support_type, :intended_stay, :created_at, prefix: "scheme", to: :scheme, allow_nil: true
+  delegate :scheme_type, to: :scheme, allow_nil: true
+
+  def scheme_code
+    scheme&.id ? "S#{scheme.id}" : nil
+  end
+
+  def scheme_owning_organisation_name
```

Comment:
These could be delegated as well couldn't they?

Reply:
Yes, though the name still has to be defined separately I think? Unless I delegated the org names to schemes and **then** to case_logs but that seems like overkill